asserts/model.go

```diff
@@ -657,6 +694,113 @@ func checkOptionalSystemUserAuthority(headers map[string]interface{}, brandID st
 	return nil, fmt.Errorf("%q header must be '*' or a list of account ids", name)
 }
 
+func checkModelValidationSetAccountID(headers map[string]interface{}, what, brandID string) (string, error) {
+	accountID, err := checkOptionalStringWhat(headers, "account-id", what)
+	if err != nil {
+		return "", err
+	}
+
+	// default to brand ID if account ID is not provided
+	if accountID == "" {
+		return brandID, nil
+	}
+	return accountID, nil
+}
+
+// checkModelValidationSetSequence reads the optional 'sequence' member, if
+// not set, returns 0 as this means unpinned. Unfortunately we are not able
+// to reuse `checkSequence` as it operates inside different parameters.
+func checkModelValidationSetSequence(headers map[string]interface{}, what string) (int, error) {
+	seq, err := checkIntWithDefaultWhat(headers, "sequence", what, 0)
```

Comment:
with this approach we let people write:

sequence: 0

but  I'm not sure we want to make that valid.

A different approach would be to check if sequence in in headers, if not return 0 otherwise return what is in sequence checking that is >=1

Reply:
I think it makes sense that we don't want `sequence: 0`, I've updated the approach to the one suggested